docs/modules/ROOT/pages/scaling-upgrading.adoc

```diff
@@ -1,6 +1,34 @@
 = Scaling and Upgrading a Hazelcast Deployment
 
-To scale an existing Hazelcast deployment, use the following command:
+To scale an existing Hazelcast deployment, there are two options:
+
```

Comment:
```suggestion
+
```

Reply:
Add this plus on this line and on each empty line related to the list items.

This is because adding a plus on empty lines that are related to a list item indents the content to make it clearer where the two options are (the list items) and what relates to that option.